tests/vectorize.py

```diff
@@ -138,6 +139,19 @@ def __call__(self, x):
         self.assertEqual(y.tolist(), [[1, 0], [3, 2], [5, 4], [7, 6], [9, 8],
                                       [10, 11], [12, 13], [15, 14], [16, 17], [18, 19]])
 
+    def test_trainvar_assign(self):
+        m = objax.ModuleList([objax.TrainVar(jn.zeros(2))])
+
+        def increase(x):
+            m[0].assign(m[0].value + 1)
+            return x + 1
+
+        x = np.arange(10)[:, None]
+        vec_increase = objax.Vectorize(increase, m.vars())
+        y = vec_increase(x)
+        self.assertEqual(y.tolist(), np.arange(1, 11)[:, None].tolist())
+        self.assertEqual(m[0].value.tolist(), [1., 1.])
```

Comment:
Similar to parallel, add another test which tests a situation when TrainVar is increased by different values (in other words test that reduce will work correctly in this case):

```
        def increase(x):
            m[0].assign(m[0].value + x)
            return x + 1

   ...
```

Reply:
Done